### ISSUE TITLE
Languages / localization: Convert Japanese site to new format and translate basic content. 

### ISSUE THREAD

**Issue by torotimes** [working-group-two/wgtwo.com#37]
Create subdomain (jp.wgtwo.com) or a structure like wgtwo.com/jp. Need to prepare our platform for multiple languages in the future, so we need to make it scaleable. Yet, important that we can make the localized pages relatively static to avoid continuous updating.

**Comment by torotimes** [comment]
Hey @s-utstrand let's start adding this to our workflow planning. Our Japanese team has submitted their idea of how we can construct a Japanese landing page, with an associated contact form. So we can take that as a starting point for how we develop both the general interaction and the individual components. 

Here's their input: https://docs.google.com/presentation/d/1Dvf_2cutOCbdqnti6jjo4WMdWx3Zc8tOGpq_SbRWC1U/edit?usp=sharing

**Comment by s-utstrand** [comment]
This sound a bit comprehensive, and it's not completely clear to me specifically what needs to be done. I suggest we schedule a meeting to discuss this.

**Comment by torotimes** [comment]
Yes, we need to meet about this one. Will invite you for a chat.

**Comment by torotimes** [comment]
Translation document: https://docs.google.com/document/d/1dJ9V3448JG-7pXw1R-ALYvkpmqGmdRsyupAhVUGggrg/edit
